enterprise/cloud.oracle/src/org/netbeans/modules/cloud/oracle/assets/ImageBuilderCommand.java

```diff
@@ -266,5 +280,24 @@ private void refresh() {
             }
         }
     }
+    
+    private static final String TotalPhysicalMemorySizeAttr = "TotalPhysicalMemorySize";    // NOI18N
+    private static final ObjectName osMXBeanName = getOperatingSystemMXBeanName();
+    
+    private int getTotalMemory() throws MBeanException, AttributeNotFoundException, InstanceNotFoundException, ReflectionException {
+
+        MBeanServer server = ManagementFactory.getPlatformMBeanServer();
+        Object totalMemory = server.getAttribute(osMXBeanName, TotalPhysicalMemorySizeAttr);
+        System.out.println("Total memory: "+totalMemory);
+        return 16;
+    }
+    
+    private static ObjectName getOperatingSystemMXBeanName() {
+        try {
+            return new ObjectName(ManagementFactory.OPERATING_SYSTEM_MXBEAN_NAME);
+        } catch (MalformedObjectNameException ex) {
+            throw new IllegalArgumentException(ex);
```

Comment:
I think  throwing IAE during static initialization will make the class unloadable and references to it will fail with a LinkageError; is it the intent ?